Tasks/VsTest/models.ts

```diff
@@ -41,6 +41,12 @@ export interface TestConfigurations {
     rerunFailedThreshold: number;
     rerunMaxAttempts: number;
     toolsInstallerConfig: ToolsInstallerConfiguration;
+
+    // Proxy properties
+    proxyUrl: string;
+    proxyUserName: string;
+    proxyPassword: string;
+    proxyBypassHosts: string;
```

Comment:
can this be a separate interface that is contained inside DtaTestConfigurations ?